### PR TITLE
update the format, command, and output of information schemas

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### What is changed, added or deleted? (Required)

Update the format, command, and output of some information schemas according to TiDB v6.5.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tidb-trx.md [hunk 1]
```diff
@@ -52,20 +51,19 @@ DESC tidb_trx;
 * `USER`：执行该事务的用户名。
 * `DB`：执行该事务的 session 当前的默认数据库名。
 * `ALL_SQL_DIGESTS`：该事务已经执行过的语句的 Digest 的列表，表示为一个 JSON 格式的字符串数组。每个事务最多记录前 50 条语句。通过 [`TIDB_DECODE_SQL_DIGESTS`](/functions-and-operators/tidb-functions.md#tidb_decode_sql_digests) 函数可以将该列的信息变换为对应的归一化 SQL 语句的列表。
+* `RELATED_TABLE_IDS`：<!--TODO-->
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-trx.md:54]: @bb7133 PTAL

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by bb7133 — APPROVED** [review]
LGTM

File: information-schema/information-schema-tidb-trx.md [hunk 2]
```diff
@@ -52,22 +53,25 @@ DESC tidb_trx;
...
```
Comment (Oreoxmt) [inline information-schema/information-schema-tidb-trx.md:56]: @wjhuang2016 PTAL

Comment (wjhuang2016) [reply information-schema/information-schema-tidb-trx.md:56]: 该事务访问的表、视图等对象的 ID。

Comment (Oreoxmt) [inline information-schema/information-schema-tidb-trx.md:56]:
```suggestion
* `RELATED_TABLE_IDS`：该事务访问的表、视图等对象的 ID。
```

**Pull request merged**